client.py

```diff
@@ -18,10 +18,12 @@
 class ClientConfig(BaseSettings):
     maxpage: int = 1000
     elasticsearch_index_name_prefix: str = ""
-    top_term_query_timeout: int = 60
+    esopts: Dict = {"request_timeout": 3600, "max_retries": 3}
```

Comment:
The thinking here was that we should let all of the requests against elasticsearch finish, ultimately- since upstream caching will still catch things if other client timeouts get triggered, but I see the wisdom in keeping the default a little more reasonable than an hour of waiting- I'll reduce to 600, we can always adjust via config later.